nucypher/blockchain/eth/actors.py

```diff
@@ -164,6 +171,12 @@ def __init__(self,
         blockchain.transacting_power.activate()
         self.log = Logger("Deployment-Actor")
 
+        # TODO: Does this class want to be a Character implementing PowerUp consumption?
```

Comment:
Would would it mean to make this change?  Moving this class into either chaotic.py or lawful.py (and deciding which one)?

Reply:
Yeah - I think into `lawful.py` - However, I'm not fully convinced.